sdk/eventgrid/Azure.Messaging.EventGrid.SystemEvents/src/CloudEventExtensions.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text.Json;
+
+namespace Azure.Messaging.EventGrid.SystemEvents
+{
+    /// <summary>
+    /// Extension methods for BinaryData to use for parsing JSON-encoded events.
+    /// </summary>
+    public static class CloudEventExtensions
```

Comment:
I decided to rename this type from the [EventGrid version](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventgrid/Azure.Messaging.EventGrid/src/EventGridExtensions.cs) and forego the type forwarding. This way we could leave everything in the SystemEvents namespace.

Reply:
Thinking about this more, I don't think we want to do this as we will end up with duplicate copies of this extension method.